mojaloop-payment-manager/Chart.yaml

```diff
@@ -25,12 +25,13 @@ dependencies:
     alias: management-api
```

Comment:
Please bump chart version.

Reply:
Thanks @mdebarros , I bumped up the major version as this is a braking change. Old helm values dont work with this version.

Reply:
And helm upgrade worked with new values